themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -118,6 +109,16 @@
           VuFind.setCurrentSearchId($searchId);
           JS;
 
+      if ($domain = $this->cookieManager()->getCookieDomain()) {
+        $appendScripts[] = 'VuFind.cookie.setDomain("' . $domain . '");';
```

Comment:
I realize this is a pre-existing issue, but while we're changing anyway, would it be a little more reliable to export these values like:

```
$appendScripts[] = 'VuFind.cookie.setDomain(' . json_encode($domain) . ');';
```

instead of manually adding quotes and leaving the value raw? There shouldn't be a need for this given "normal" configured values, but this might prevent some weirdness with bad configurations.

Reply:
done